crates/medmodels-core/src/medrecord/advanced_example_dataset/synthetic_data/README.md

```diff
@@ -0,0 +1,82 @@
+# Attribution
+
+This dataset was created using the [Synthea™ Patient Generator](https://github.com/synthetichealth/synthea).
+
+## Dataset Description
+
+The dataset is a synthetic dataset created to simulate a medium-scale medical record system containing comprehensive patient data. The dataset encompasses details for six hundred hypothetical patients, capturing key medical attributes including patient demographics, diagnosis, medical procedures undertaken, medications prescribed and whether any of them have died and when.
+
+## Data Schema Nodes
+
+### 1. `patient.csv`
+
+The patient data consists of 600 hypothetical patients.
+
+- **patient_id:** A unique identifier assigned each patient
+- **gender:** Recorded as Male, Female, or Non-Binary
+- **age:** Patient's age in years
+
+### 2. `diagnosis.csv`
+
+There are 608 unique diagnosis in the dataset.
+
+- **diagnosis_code:** Unique code identifiying a specific diagnosis
+- **description:** Description of the diagnosis
+
+### 3. `procedure.csv`
+
+The procedure data contains 163 different medical procedures.
+
+- **procedure_code:** Unique identifier for each procedure
+- **description:** Detailed procedural information
+
+### 4. `drug.csv`
+
+The drug data has 43 different prescribed medications.
+
+- **drug_code:** Unique identifier for the medication prescribed
+- **description:** Generic name of the drug and dosage
+
+### 5. `event.csv`
+
+The drug data has 1 possible event
+
+- **event:** Event name (e.g., death)
+
+## Data Schema Edges
+
+### 1. `patient_diagnosis.csv`
+
+There are 5741 diagnosis instances for the 600 patients.
+
+- **patient_id:** Linked to the Patient Demographics
```

Comment:
```suggestion
- **patient_id:** Linked to the Patient demographic
```

(I realised this error is coming from my description, but demographics here is a bit weird)

Reply:
yes, maybe change it to `Unique identifier assigned each patient` better